ash-multi

```diff
@@ -357,6 +359,10 @@ while (("$#")); do
     ASH_DEBUG_ENABLED="true"
     ASH_SCANSET_ARGS="${ASH_SCANSET_ARGS} --debug"
     ;;
+  --format)
+    shift
+    ASH_OUTPUT_FORMAT="$1"
```

Comment:
I think there is still a code path where this is "Used before set".  If `ash-multi` is run by running the container image directly (without using the `ash` wrapper script to run the `${OCI_RUNNER} run ...` command), then the environment variable `ASH_OUTPUT_FORMAT` may NOT get set.  (It's not getting set in `Dockerfile`).

I think that `ash-multi` should also initialize this value - around line 283.